#!/usr/bin/env python

PACKAGE = 'pointcloud_to_laserscan'

from dynamic_reconfigure.parameter_generator_catkin import ParameterGenerator, str_t, double_t, bool_t, int_t

gen = ParameterGenerator()
# bounding box
gen.add("range_x_max", double_t, 0, "The maximum range along the x-axis in the robot's coordinate system.", 0, -5.0, 5.0)
gen.add("range_x_min", double_t, 0, "The minmum range along the x-axis in the robot's coordinate system.", 0, -5.0, 5.0)
gen.add("range_y_max", double_t, 0, "The maximum range along the y-axis in the robot's coordinate system.", 0, -5.0, 5.0)
gen.add("range_y_min", double_t, 0, "The minimum range along the y-axis in the robot's coordinate system.", 0, -5.0, 5.0)

exit(gen.generate(PACKAGE, "pointcloud_to_laserscan_node", "RobotFootprint"))
